modules/ytp-assets-common/src/less/helpers.less

```diff
@@ -0,0 +1,9 @@
+// helper classes to use in html
+
+.flex-1 {
+  .flex(1 1 0) !important;
```

Comment:
!important probably shouldn't be used and in the future it might be denied in linting. https://github.com/lesshint/lesshint/blob/master/lib/linters/README.md#importantrule

Reply:
I added this to add a few helper classes that can be plugged in html to get the wanted behaviour. In a similar way that bootstrap has utility classes, eg. https://getbootstrap.com/docs/4.0/utilities/flex/ .

Here's a discussion about why for example bootstrap uses important: https://stackoverflow.com/a/35710056/6931954 .

Should I add an exception to the linting for this one, or what do you think about this? I think utility classes would help alot in keeping the css styles more managable.

Reply:
I added the exception for linting. It now ignores the !important rule for those lines.